enos/enos.py

```diff
@@ -577,18 +579,39 @@ def tc(env=None, **kwargs):
 
 
 @enostask("""
-usage: enos info [-e ENV|--env=ENV]
-
+usage: enos info [-e ENV|--env=ENV] [--out={json,yaml}]
 
 Show information of the `ENV` deployment.
 
 Options:
   -e ENV --env=ENV     Path to the environment directory. You should
                        use this option when you want to link a specific
                        experiment [default: %s].
+  --out {json|yaml}    Output the result in either json or yaml format.
 """ % SYMLINK_NAME)
 def info(env=None, **kwargs):
-    pprint.pprint(env)
+    def host_to_json(o):
+        if isinstance(o, Host):
+            return {
```

Comment:
Unfortunately it's not so simple. To use `__dict__` you have to implement it into `Host.py`. But, if you do so, the yaml file will use this method rather than the really useful actual one that dump Host info together with a link to the class `Host.py`. As a result, you have to write a custom `yaml.load(config)` function that tells yaml how to parse host values. From my point of view, the impact of implementing `__dict__` in `Host.py` results in much wider implication than implementing this lambda.